equinix_metal.tf

```diff
@@ -21,44 +21,44 @@ data "template_file" "user_data" {
   }
 }
 
-resource "metal_project" "new_project" {
-  count           = var.metal_create_project ? 1 : 0
+resource "equinix_project" "new_project" {
```

Comment:
This should be [`equinix_metal_project`](https://registry.terraform.io/providers/equinix/equinix/latest/docs/resources/equinix_metal_project)